src/Dependencies/Collections/SegmentedArray`1.cs

```diff
@@ -429,8 +429,6 @@ public TestAccessor(SegmentedArray<T> array)
             }
 
             public static int SegmentSize => SegmentedArray<T>.SegmentSize;
-
-            public T[][] Items => _array._items;
```

Comment:
>Items [](http://example.com/codeflow?start=25&length=5)

Does this class still get constructed anywhere?

Reply:
It does not, so I'll simplify further.